apps/cron/src/tasks/processGemsPayout/index.ts

```diff
@@ -82,7 +83,7 @@ export async function processGemsPayout(ctx: Context, { now = DateTime.utc() }:
       proofsMap: {},
       season,
       claims: [],
-      totalClaimable: totalPoints,
+      totalClaimable: (BigInt(Math.floor(totalPoints)) * BigInt(10 ** scoutTokenDecimals)).toString(),
```

Comment:
are weeklyClaims even used in prod? it seems like we are creating them - btu there's only 10 records in the DB